src/olympia/blocklist/templates/blocklist/includes/logs.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+
+<ul>
+    {% for log in logs %}
+      <li>
+        {% if log.details.url %}
+            <a href="{{ log.details.url }}">{{ log.created|date:"Y-m-d" }}</a>.
+        {% else %}
+            {{ log.created|date:"Y-m-d" }}.
+        {% endif %}
+        {{ log.log.short }} by {{ log.author_name }}:
+        {% if log.details %}{{ log.details.guid }}{% else %}{{ log.arguments.1 }}{% endif %}{% if 'min_version' in log.details %}
+            , {% trans 'versions' %} {{ log.details.min_version }} - {{ log.details.max_version }}.
+        {% else %}.
+        {% endif %}
+        {% if log.details.include_in_legacy %}
+            {% trans 'Included in legacy blocklist.' %}
+        {% endif %}
+        <ul>
+            <li>{{ log.details.reason }}</li>
+        </ul>
+      </li>
+    {% endfor %}
+    {{ addtional_content }}
```

Comment:
How about moving the `format_html()` call generating this to the template as well, for consistency ? I think you only need to pass the `submission` instance to do this.

Reply:
I guess I could too.  (I was only thinking about the increasingly convoluted code for the Block logs.)

Reply:
Yeah re-reading this, especially because it's inside a `<ul>` I think it's best to move everything to the template. There is no good reason to do this outside now.

Reply:
https://github.com/mozilla/addons-server/pull/13525/commits/29a4847976c79bd14ad0d93f7c78bdcb530ba66e